controllers/controllers.go

```diff
@@ -42,6 +42,17 @@ const (
 	podSchedulingDelayDuration = 15 * time.Second
 )
 
+// containsAll determines if one map contains all the keys and matching values
+// from another map.
+func containsAll(current map[string]string, desired map[string]string) bool {
```

Comment:
I believe a better place would be the internal package maybe a new `map_helper` file?